content/guides/models/track/limits.md

```diff
@@ -149,10 +149,12 @@ with f as open("large_config.json", "r"):
     large_config = json.load(f)
     wandb.init(config=large_config)
 ```
+## Workspace considerations 
+
 
 ### Run count
 
-For faster loading times, keep the total number of runs in a single project under 10,000. Large run counts can slow down project workspaces and runs table operations, especially when grouping is enabled or runs have a large count of distinct metrics.
+For fast loading times, keep the total number of runs in a single project under 100,000 (10,000 on Dedicated Cloud or Self-managed W&B deployments). Large run counts can slow down project workspaces and runs table operations, especially when grouping is enabled or runs have a large count of distinct metrics.
```

Comment:
```suggestion
To reduce loading times, keep the total number of runs in a single project under:

- 100,000 on SaaS Cloud
- 10,000 on Dedicated Cloud or Self-managed

Run counts over these thresholds can slow down operations that involve project workspaces or runs tables, especially when grouping runs or collecting a large number of distinct metrics during runs. See also the [Metric count]({{< relref "#metric-count" >}}) section.
```
Reducing some opinions here -- runs might be slow for other reasons. Also making the specific recommendations easier to spot. Please edit the edit if it is introducing some inaccuracy, though.